helm/charts/keto/templates/NOTES.txt

```diff
@@ -1,21 +1,20 @@
 1. Get the application URL by running these commands:
-{{- if .Values.ingress.enabled }}
-{{- range $host := .Values.ingress.hosts }}
+{{- if .Values.ingress.read.enabled }}
+{{- range $host := .Values.ingress.read.hosts }}
```

Comment:
Should we add write as well? There are now two endpoints instead of just one :thinking: